__tests__/app.test.js

```diff
@@ -162,7 +162,83 @@ describe('GET /api/articles/:article_id/comments', () => {
         })
     })
   })
-  
+
+
+describe("POST: /api/articles/2/comments", () => {
+    test("status 201: should post a comment for an article and respond with the posted comment", () => {
+        const data = {
+            body: "This is a great article",
+            username: "butter_bridge"
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(data)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.comment).toMatchObject({
```

Comment:
We could assert the comment has the values we sent, present on the correct properties here.